src/pages/Environments.js

```diff
@@ -116,8 +120,176 @@ const DeleteAppModal = ({ app: { googleProject, appName, diskName, appType }, on
+    }
+
+    return div({ style: { ...props?.style, display: 'flex', flexDirection: 'row', alignItems: 'center' } }, [
+      h(LabeledCheckbox, { checked: deleteDisk, onChange }, []),
+      span({ style: { paddingLeft: '0.5rem' } }, ['Delete this disk.'])
+    ])
+  }
+
+  const costPerCopy = getPersistentDiskCostMonthly(disk, getRegionFromZone(disk.zone))
+  const numberOfCopies = _.flow(_.values, _.compact, _.size)(isWorkspaceSelected)
+
+  const space = { style: { marginTop: '0.5rem', marginBottom: '0.5rem' } }
+
+  return h(Modal, {
+    title: `Migrate ${disk.name}`,
+    okButton: deleteDisk || _.isEmpty(workspaces) ?
+      h(ButtonPrimary, { disabled: !deleteDisk, onClick: _.flow(onDismiss, onDeleteDisk) }, 'Delete') :
+      h(ButtonPrimary, { disabled: numberOfCopies === 0, onClick: migrateDisk }, 'Migrate'),
+    onDismiss
+  }, [
+    div({ style: { display: 'flex', flexDirection: 'column' } }, Array.prototype.concat(
+      [span(space, 'Due to data security policies, persistent disks can no longer be shared between workspaces.')],
+      _.isEmpty(workspaces) ? [
+        strong(space, ['You own this disk but do not have access to any workspaces where it can be shared.']),
+        renderDeleteDiskSelection(space),
+        span(space, 'OR'),
+        div([
+          h(Link, { onClick: _.flow(onDismiss, onContactSupport) }, 'Contact Terra Support'),
+          ' to have it transferred to another user.'
+        ])
+      ] : [
+        strong(space, ['Select workspaces where you what to use a copy of this disk.']),
+        h(FlexTable, {
+          'aria-label': 'workspace-selection',
+          width: 400, height: 250, headerHeight: 24, rowHeight: 24, variant: 'light',
+          rowCount: _.size(workspaces),
+          columns: [
+            {
+              field: 'selection',
+              size: { basis: 24, grow: 0 },
+              headerRenderer: () => h(TooltipTrigger, { content: 'Select all' }, [
+                h(LabeledCheckbox, {
+                  checked: _.every(w => isWorkspaceSelected[w.workspaceId], workspaces),
+                  onChange: selected => {
+                    setDeleteDisk(false)
+                    setIsWorkspaceSelected(_.reduce((state, w) => _.set(w.workspaceId, selected, state), {}, workspaces))
+                  }
+                }, [])
+              ]),
+              cellRenderer: ({ rowIndex }) => {
+                const workspaceId = workspaces[rowIndex].workspaceId
+                return h(LabeledCheckbox, {
+                  checked: isWorkspaceSelected[workspaceId],
+                  onChange: selected => {
+                    setIsWorkspaceSelected(_.set(workspaceId, selected, isWorkspaceSelected))
+                    setDeleteDisk(false)
+                  }
+                })
+              }
+            },
+            {
+              field: 'workspace',
+              headerRenderer: () => strong(['Workspace name']),
```

Comment:
Maybe use the `HeaderCell` component from the table module instead of strong here. [strong](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong) doesn't seem really appropriate here and that would make the style consistent with other tables.